util/all_tests.json

```diff
@@ -1,49 +1,61 @@
 [
   {
-    "cmd": ["crypto/crypto_test"]
+    "cmd": ["crypto/crypto_test"],
+    "valgrind_supp": ["valgrind_suppressions_crypto_test.supp"]
   },
   {
-    "cmd": ["crypto/crypto_test", "--gtest_also_run_disabled_tests", "--gtest_filter=BNTest.DISABLED_WycheproofPrimality"]
+    "cmd": ["crypto/crypto_test", "--gtest_also_run_disabled_tests", "--gtest_filter=BNTest.DISABLED_WycheproofPrimality"],
+    "skip_valgrind": true
   },
   {
     "cmd": ["crypto/crypto_test", "--gtest_also_run_disabled_tests", "--gtest_filter=RSATest.DISABLED_BlindingCacheConcurrency"],
-    "skip_sde": true
+    "skip_sde": true,
+    "skip_valgrind": true
   },
   {
-    "cmd": ["crypto/urandom_test"]
+    "cmd": ["crypto/urandom_test"],
+    "skip_valgrind": true
   },
   {
     "comment": "Without RDRAND",
     "cmd": ["crypto/urandom_test"],
-    "env": ["OPENSSL_ia32cap=~0x4000000000000000"]
+    "env": ["OPENSSL_ia32cap=~0x4000000000000000"],
+    "skip_valgrind": true
```

Comment:
Why these tests are disabled?

> The only unit test suite that is run is crypto_test. The reason is that it takes +1 hour for only this test suite.

Which instance type was used to test and result in `1 hour`? I enabled the flag `useValgrind` in place and the `all_tests.go` finished within 15 min. The instance I used is [BUILD_GENERAL1_LARGE](https://docs.aws.amazon.com/codebuild/latest/userguide/build-env-ref-compute-types.html).

Reply:
You can see in the CI, that running this one test took 25 mins. Maybe your tests failed somehow without reporting.

We can start small and enable tests as we go when we are more confident test time does not explode.